src/components/LinkChooser/LinkChooser.tsx

```diff
@@ -195,6 +196,20 @@ export const LinkChooser: FC<LinkChooserProps> = ({
         autoResize: true,
     });
 
+    const {
+        isOpen,
+        selectionManager: { focusedKey },
+    } = state;
+
+    useEffect(() => {
+        if (focusedKey && popoverRef.current && isOpen) {
+            const dropdownItem = popoverRef.current?.querySelector(`[data-key="${focusedKey}"]`);
```

Comment:
```suggestion
            const dropdownItem = popoverRef.current.querySelector(`[data-key="${focusedKey}"]`);
```

Reply:
This causes a typeError, hence the check afterwards for "if dropdownItem" :)

Reply:
Oh wait, I'm lying, nevermind!

Reply:
fixed :)